docs/updates/2019-02.md

```diff
@@ -0,0 +1,93 @@
+# February 2019
+
+Welcome to the February 2019 Qlik Core update. We're highlighting a couple of important changes and
+improvements which we hope you will like.
+
+## Usability and design changes in catwalk
+
+Since being moved to stable last month, we've made several small usability improvements and design changes along with a
+couple of bug fixes.
+
+### QlikView select excluded coloring
+
+We have added support for the *select excluded* selection state, which can be used for deeper data model analysis. A
+value enters the selected excluded state because the value was first selected, and then excluded by a selection in
+another field. This value will be described in catwalk with *green on light grey background*.
+
+!!! Note
+    Please note the color scheme follows QlikView coloring and corresponds to *dark grey and check mark* in Qlik
+    Sense.
+
```

Comment:
IMO, remove the note.

Reply:
Think so too, I like how they are readable on GitHub (which obviously doesn't support `!!! Note`): https://github.com/qlik-oss/core-website/blob/feb-update/docs/updates/2019-02.md

Perhaps we can also state that it's _QlikView inspired_, not necessarily 1:1 mapping of it?

Reply:
👍